docs/src/content/docs/tr/guides/authoring-content.md

```diff
@@ -139,7 +139,7 @@ Uyarı ve tehlike ara bölümleri, kullanıcıların dikkatini gözden kaçabile
 Bunları çok kullandığınızı farkederseniz, dokümanınızın yeniden oluşturulmasına gerek kalmayacağının işareti olabilir.
 
 :::caution[Uyarı]
-Harika bir dokümantasyon sitesi istediğine emin değilsen, [Starlight](../../) kullanmadan önce iki kez düşün.
+Harika bir dokümantasyon sitesi istediğine emin değilsen, [Starlight](../../../) kullanmadan önce iki kez düşün.
```

Comment:
I think this was fine as it was — otherwise this link navigates back up to the English homepage instead of the Turkish one.

```suggestion
Harika bir dokümantasyon sitesi istediğine emin değilsen, [Starlight](../../) kullanmadan önce iki kez düşün.
```